gocov/test.go

```diff
@@ -45,3 +146,66 @@ func runTests(args []string) error {
 	}
 	return convertProfiles(coverprofile.Name())
 }
+
+func runAllTests(args []string) (err error) {
+	pkgRoot, _ := os.Getwd()
+	pkg, err2 := relToGOPATH(pkgRoot)
+	if err2 != nil {
+		return err2
+	}
+	// TODO(maruel): This assumes this starts with "./". This is
+	// incorrect,someone could request to run test in a separate package.
+	requestedPath := filepath.Join(pkgRoot, args[0][:len(args[0])-3])
+	testDirs := goTestDirs(requestedPath)
+	if len(testDirs) == 0 {
+		return nil
+	}
+
+	tmpDir, err2 := ioutil.TempDir("", "gocov")
+	if err2 != nil {
+		return err2
+	}
+	defer func() {
+		err2 := os.RemoveAll(tmpDir)
+		if err == nil {
+			err = err2
+		}
+	}()
+
+	// It passes a unique -coverprofile file name, so that all the files can
+	// later be merged into a single file.
+	var wg sync.WaitGroup
+	results := make(chan *result, len(testDirs))
+	for i, td := range testDirs {
+		wg.Add(1)
+		go func(index int, testDir string) {
+			defer wg.Done()
+			args := []string{
+				"go", "test", "-covermode=count", "-coverpkg", pkg + "/...",
```

Comment:
Why specify covermode? It's not done for single package, I don't see why it should be done here.